src/App.js

```diff
@@ -31,29 +24,39 @@ const App = () => (
       <GridRow>
         <GridCol>
           <Field
-            name="dob"
-            component={DateInputAdapter}
+            name="firstName"
+            component={InputField}
             validate={required}
-            hintText="Date of birth"
-            labelPosition="right"
-          />
+          >
+            First name
+          </Field>
         </GridCol>
       </GridRow>
       <GridRow>
         <GridCol>
+          <Field
+            name="description"
+            component={TextArea}
+            validate={required}
+            hint="Any other information you want to provide"
+          >
+            Description
+          </Field>
+        </GridCol>
+      </GridRow>
+      <GridRow>
+        <GridCol columnOneQuarter>
           <Button type="submit" disabled={submitting}>
-            Submit
+            Log In
           </Button>
         </GridCol>
-        <GridCol>
-          <Button
-            type="button"
-            onClick={reset}
-            disabled={submitting || pristine}
-          >
+        <GridCol columnOneQuarter>
+          <Button onClick={reset} disabled={submitting || pristine}>
             Reset
           </Button>
         </GridCol>
+        <GridCol columnOneQuarter />
```

Comment:
do we really need these empty grid bits? Not great for example code to have useless markup